Auto update failing for neovim-nightly

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the neovim-nightly package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating neovim-nightly [Current version: 0.8.0-dev+504-g7e1cf6b76]
INFO: package being updated to 0.8.0-dev+513-g01fc5097d.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 4917k    0 4917k    0     0  3120k      0 --:--:--  0:00:01 --:--:-- 4932k
100 10.4M    0 10.4M    0     0  4150k      0 --:--:--  0:00:02 --:--:-- 5354k
100 10.5M    0 10.5M    0     0  4142k      0 --:--:--  0:00:02 --:--:-- 5322k

You are about to commit these changes:

--------------------
diff --git a/packages/neovim-nightly/build.sh b/packages/neovim-nightly/build.sh
index 414c638a2..1fedb4d75 100644
--- a/packages/neovim-nightly/build.sh
+++ b/packages/neovim-nightly/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://neovim.io
 TERMUX_PKG_DESCRIPTION="Ambitious Vim-fork focused on extensibility and agility (nvim-nightly)"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Aditya Alok <alok@termux.org>"
-TERMUX_PKG_VERSION="0.8.0-dev+504-g7e1cf6b76"
+TERMUX_PKG_VERSION="0.8.0-dev+513-g01fc5097d"
 TERMUX_PKG_SRCURL="https://github.com/neovim/neovim/archive/nightly.tar.gz"
-TERMUX_PKG_SHA256=3a87572e4f578e781c1cb146c336d7ef80f0039a6430eb5496a2eaec730aec47
+TERMUX_PKG_SHA256=47f4d5fab8d173c87727375e148e1801bed8650ed4d034bac2ac7e9192123086
 TERMUX_PKG_DEPENDS="libiconv, libuv, luv, libmsgpack, libandroid-support, libvterm, libtermkey, libluajit, libunibilium, libtreesitter"
 TERMUX_PKG_HOSTBUILD=true
 
--------------------

neovim-nightly: update to 0.8.0-dev+513-g01fc5097d


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'termux/package-builder'...
termux - building neovim-nightly for arch aarch64...
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13134  100 13134    0     0  16773      0 --:--:-- --:--:-- --:--:-- 16773
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100   833  100   833    0     0   6080      0 --:--:-- --:--:-- --:--:--  6080
gpg: Signature made Wed 29 Jun 2022 05:40:06 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13320  100 13320    0     0  73591      0 --:--:-- --:--:-- --:--:-- 73591
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100   833  100   833    0     0   5950      0 --:--:-- --:--:-- --:--:--  5950
gpg: Signature made Sun 26 Jun 2022 08:39:34 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13120  100 13120    0     0  53333      0 --:--:-- --:--:-- --:--:-- 53333
100 13120  100 13120    0     0  53333      0 --:--:-- --:--:-- --:--:--     0
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100   833  100   833    0     0   5907      0 --:--:-- --:--:-- --:--:--  5907
gpg: Signature made Fri 24 Jun 2022 10:11:18 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading dependency libandroid-support@28-2 if necessary...
Downloading dependency libiconv@1.16-3 if necessary...
Downloading dependency iconv@1.16-3 if necessary...
Downloading dependency libluajit@1:2.1.0-beta3-3 if necessary...
Found libluajit in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libl/libluajit/libluajit_1:2.1.0-beta3-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  192k  100  192k    0     0   796k      0 --:--:-- --:--:-- --:--:--  796k
extracting libluajit...
Downloading dependency luajit@1:2.1.0-beta3-3 if necessary...
Found luajit in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/l/luajit/luajit_1:2.1.0-beta3-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  229k  100  229k    0     0  1103k      0 --:--:-- --:--:-- --:--:-- 1103k
extracting luajit...
Downloading dependency libmsgpack@4.0.0 if necessary...
Found libmsgpack in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libm/libmsgpack/libmsgpack_4.0.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 24148  100 24148    0     0  51052      0 --:--:-- --:--:-- --:--:-- 51052
extracting libmsgpack...
Downloading dependency libunibilium@2.1.1 if necessary...
Found libunibilium in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libu/libunibilium/libunibilium_2.1.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 19450  100 19450    0     0  89631      0 --:--:-- --:--:-- --:--:-- 89631
100 19450  100 19450    0     0  89631      0 --:--:-- --:--:-- --:--:--     0
extracting libunibilium...
Downloading dependency libtermkey@0.22-3 if necessary...
Found libtermkey in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libt/libtermkey/libtermkey_0.22-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 16228  100 16228    0     0   111k      0 --:--:-- --:--:-- --:--:--  111k
extracting libtermkey...
Downloading dependency libtreesitter@0.20.6 if necessary...
Found libtreesitter in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libt/libtreesitter/libtreesitter_0.20.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 62160  100 62160    0     0   260k      0 --:--:-- --:--:-- --:--:--  260k
extracting libtreesitter...
Downloading dependency libuv@1.44.1 if necessary...
Found libuv in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libu/libuv/libuv_1.44.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 73168  100 73168    0     0   297k      0 --:--:-- --:--:-- --:--:--  297k
extracting libuv...
Downloading dependency libvterm@19.09.17-1 if necessary...
Found libvterm in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/libv/libvterm/libvterm_19.09.17-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 35454  100 35454    0     0   233k      0 --:--:-- --:--:-- --:--:--  372k
extracting libvterm...
Downloading dependency luv@1.43.0-0-0 if necessary...
Found luv in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/l/luv/luv_1.43.0-0-0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 41444  100 41444    0     0   179k      0 --:--:-- --:--:-- --:--:--  179k
extracting luv...
Downloading https://github.com/neovim/neovim/archive/nightly.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13205    0 13205    0     0  20378      0 --:--:-- --:--:-- --:--:-- 20378
100 5488k    0 5488k    0     0  3517k      0 --:--:--  0:00:01 --:--:-- 5996k
100 10.5M    0 10.5M    0     0  4371k      0 --:--:--  0:00:02 --:--:-- 5910k
Applying patch: *.patch.beforehostbuild
Downloading https://cmake.org/files/v3.23/cmake-3.23.2-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 43.8M    0  208k    0     0   338k      0  0:02:12 --:--:--  0:02:12  337k
 65 43.8M   65 28.6M    0     0  18.0M      0  0:00:02  0:00:01  0:00:01 18.0M
100 43.8M  100 43.8M    0     0  22.0M      0  0:00:01  0:00:01 --:--:-- 22.0M
CMake Error: The source directory "/home/builder/.termux-build/neovim-nightly/src/third-party" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-06-29 06:19:44 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/2581035176">2581035176</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/neovim-nightly/build.sh
+++ b/packages/neovim-nightly/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://neovim.io
 TERMUX_PKG_DESCRIPTION="Ambitious Vim-fork focused on extensibility and agility (nvim-nightly)"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Aditya Alok <alok@termux.org>"
-TERMUX_PKG_VERSION="0.8.0-dev+504-g7e1cf6b76"
+TERMUX_PKG_VERSION="0.8.0-dev+513-g01fc5097d"
 TERMUX_PKG_SRCURL="https://github.com/neovim/neovim/archive/nightly.tar.gz"
-TERMUX_PKG_SHA256=3a87572e4f578e781c1cb146c336d7ef80f0039a6430eb5496a2eaec730aec47
+TERMUX_PKG_SHA256=47f4d5fab8d173c87727375e148e1801bed8650ed4d034bac2ac7e9192123086
 TERMUX_PKG_DEPENDS="libiconv, libuv, luv, libmsgpack, libandroid-support, libvterm, libtermkey, libluajit, libunibilium, libtreesitter"
 TERMUX_PKG_HOSTBUILD=true
 
@@ -91,7 +91,7 @@ termux_step_host_build() {
 
 	mkdir -p $TERMUX_PKG_HOSTBUILD_DIR/deps
 	cd $TERMUX_PKG_HOSTBUILD_DIR/deps
-	cmake $TERMUX_PKG_SRCDIR/third-party
+	cmake $TERMUX_PKG_SRCDIR/cmake.deps
 
 	make -j 1 ||
 		(_patch_luv $TERMUX_PKG_HOSTBUILD_DIR/deps && make -j 1)

